src/app/features/collectibles/components/collectible-hover.tsx

```diff
@@ -39,21 +39,17 @@ export function CollectibleHover({
       </Box>
       {onClickCallToAction && (
         <IconButton
+          height="32px"
+          width="32px"
           _focus={{ outline: 'focus' }}
+          borderRadius="xs"
           alignItems="center"
-          bg="ink.background-primary"
-          borderRadius="50%"
           display="flex"
-          height="30px"
           justifyContent="center"
           icon={
-            <ArrowUpIcon
-              transform="rotate(45deg)"
-              borderRadius="50%"
-              width="30px"
-              height="30px"
-              _hover={{ bg: 'ink.component-background-hover' }}
-            />
+            <styled.div bg="ink.background-primary" padding="space.01">
```

Comment:
The padding `space.01` is needed so that the button looks right however this means on hover there is a visible gap and difference in colour. 

If I apply the `_hover={{ bg: 'ink.component-background-hover' }}` to this wrapper then in light mode when you hover the button colour makes it look transparent as per https://github.com/leather-io/extension/issues/4971 . 

I tried a few things here including negative margin but can't figure it out yet